spec/spec_helper.rb

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
+$LOAD_PATH.unshift("#{__dir__}/../tasks")
```

Comment:
Best not mess with `$LOAD_PATH`. In my spec I simply use `require_relative`

Reply:
I had that first but it was about 5 layers of `../`. Don't mind doing that though if desired!